internal/pkg/cli/validate.go

```diff
@@ -45,6 +52,13 @@ func validateSvcPort(val interface{}) error {
 	}
 	return nil
 }
+func prettify(inputStrings []string) string {
```

Comment:
is this the same function as `quoteAll`?

Reply:
Almost. Moved a call to QuoteAll into this prettify function to streamline it.